clang/test/CodeGen/RISCV/attr-rvv-vector-bits-types.c

```diff
@@ -572,1483 +1121,44 @@ void f() {
 //===----------------------------------------------------------------------===//
 // Structs and unions
 //===----------------------------------------------------------------------===//
-// CHECK-64:      %struct.struct_int8m1 = type { <8 x i8> }
```

Comment:
Using the script erased a large portion of the CHECKs in this test. The script only knows how to check function bodies. It doesn't know about global variables or types.

Reply:
Done